IdentityCore/src/controllers/broker/MSIDSSOExtensionInteractiveTokenRequestController.m

```diff
@@ -32,6 +32,8 @@
 
 @implementation MSIDSSOExtensionInteractiveTokenRequestController
 
+NSString* const MSIDExpectedErrorDomain = @"ExpectedErrorDomain";
```

Comment:
Why do we need a separate domain for this error? This flow seems to be similar to MSIDErrorServerProtectionPoliciesRequired error